tools/ci/enforce_ogg_sample_rate.sh

```diff
@@ -0,0 +1,10 @@
+for file in $(find . -type f -name '*.ogg'); do
+  sample_rate=$(ffprobe -v error -select_streams a:0 -show_entries stream=sample_rate -of default=noprint_wrappers=1:nokey=1 "$file")
+  if [ "$sample_rate" != "44100" ]; then
+    fail = true
+    echo "Error: $file has sample rate $sample_rate Hz (expected 44100 Hz)"
```

Comment:
You may want to explain what this means and how someone can fix it in this message

Reply:
https://github.com/tgstation/tgstation/pull/87686/commits/b6defb328cc64fa47a3717fe59e980a18df0a4b2